src/borgstore/backends/rclone.py

```diff
@@ -48,9 +49,11 @@ def get_rclone_backend(url):
     try:
         info = json.loads(subprocess.check_output([RCLONE, "rc", "--loopback", "core/version"]))
     except Exception:
-        raise BackendDoesNotExist("rclone binary not found on the path or not working properly")
+        rclone_log.error("rclone binary not found on the path or not working properly")
+        return None
     if info["decomposed"] < [1, 57, 0]:
-        raise BackendDoesNotExist(f"rclone binary too old - need at least version v1.57.0 - found {info['version']}")
+        rclone_log.error(f"rclone binary too old - need at least version v1.57.0 - found {info['version']}")
+        return None
     rclone_regex = r"""
```

Comment:
Maybe it should FIRST look at the url (parse it with the regex) and only if the url matches (-> user wants rclone!), then it should invoke rclone, check the version, etc. If rclone is not wanted, the regex won't match and we can return None.

Reply:
Agreed, will make the changes.